cmd/scheduleapi/handler/handler.coverprofile

```diff
@@ -0,0 +1,62 @@
+mode: atomic
```

Comment:
`.gitignore`!

Reply:
nitpick: good catches on both of these things